typescript-test-samples/lambda-handler-dynamodb/lib/lambda-handler-dynamodb-stack.ts

```diff
@@ -0,0 +1,39 @@
+import * as cdk from 'aws-cdk-lib';
```

Comment:
Add copyright lines at the top of this file (see /src/app.ts for example)

Sorry, missed this one on the last review.